hdinsight-genomics-adam-latest-build/readme.md

```diff
@@ -0,0 +1,7 @@
+# Create HDInsight Linux cluster running the genomics analysis platform called ADAM
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FExchMaster%2Fazure-quickstart-templates%2Fmaster%2Fhdinsight-genomics-adam-latest-build%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+
+This template creates an HDInsight Linux based cluster and then updates the cluster headnodes with the genomics analysis platoform ADAM.  It also deploys Apache Spark 1.4.1 with YARN in support of ADAM.<br>
```

Comment:
Can you add more description about the genomics analysys platform? also links for more information would be helpful